schema_utils.py

```diff
@@ -236,8 +236,10 @@ def has_length_at_most(obj, max_value):
 
     @staticmethod
     def is_nonempty(obj):
-        """Returns True iff the given object (a string) is nonempty."""
-        return bool(obj)
+        """Always returns True so that an input field can be made required
```

Comment:
No, the function should return True if it's nonempty and False otherwise. We need to preserve the behaviour in order to ensure that user-submitted values are valid.

I don't think the line you're pointing to is the right one -- that just checks the keys, and doesn't apply any validators. Is it schema_utils.normalize_against_schema() that's the issue here? As part of the function, validation is applied.

If so, my question is: what is the code that's responsible for passing the image UI config to be validated here? Can you post the full traceback of the error?